powermax/provider/volume_resource.go

```diff
@@ -77,6 +78,11 @@ func (r volumeResource) Schema(ctx context.Context, req resource.SchemaRequest,
 				Required:            true,
 				Validators: []validator.String{
 					stringvalidator.LengthAtLeast(1),
+					stringvalidator.LengthAtMost(64),
```

Comment:
@doriac11 can you update description also like we did for all others?

Reply:
Ahh yes I can

Reply:
Done